app/views/users/edit.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="user-edit-page">
+
+  <div class="details tab-content">
+    <%= form_with model: current_user, local: true, html: {class: "form-horizontal form-edit-user"} do |f| %>
+      <div class="form-group row">
+        <div class="col-sm-3 col-form-label">
+          <% if current_user.avatar.attached? %>
+            <%= image_tag current_user.avatar, class: 'avatar' %>
+          <% end %>
+        </div>
+        <div class="col-sm-9">
+          <p class="username"><%=current_user.username %></p>
+        </div>
+   
+      <div class="form-group row">
+        <%= f.label :profile_picture, class: 'col-sm-3 col-form-label' %>
+        <div class="col-sm-9">
+          <%= f.file_field :avatar, class: 'form-control' %>
```

Comment:
Form currently accepts files that are not image files. Change it so that these files cannot be uploaded or are deleted upon upload. Reference the gem used for the same function in posts.

Reply:
done!!